constants/menuLinks.ts

```diff
@@ -184,15 +183,14 @@ const FOOTER_ELEMENTS: FooterStructure = [
       QISKIT_VIDEOS
     ]
   },
-  // Elements in the fourth footer column
   {
     title: 'Community',
     elements: [
       COMMUNITY_SUB_LINKS.events,
       QISKIT_ADVOCATES.advocates
+      // QISKIT_CAREERS
```

Comment:
Non-blocking: Add a TODO referring to issue #663 like, "enable when #663 is done" or something like this.